.github/actions/next-stats-action/src/prepare/repo-setup.js

```diff
@@ -53,93 +54,129 @@ module.exports = (actionInfo) => {
         }
       }
     },
-    async linkPackages(repoDir = '', nextSwcPkg) {
-      const pkgPaths = new Map()
-      const pkgDatas = new Map()
-      let pkgs
+    async linkPackages({ repoDir = '', nextSwcPkg, parentSpan }) {
+      const rootSpan = parentSpan
+        ? parentSpan.traceChild('linkPackages')
+        : trace('linkPachages')
 
-      try {
-        pkgs = await fs.readdir(path.join(repoDir, 'packages'))
-      } catch (err) {
-        if (err.code === 'ENOENT') {
-          console.log('no packages to link')
-          return pkgPaths
+      return await rootSpan.traceAsyncFn(async (rootSpan) => {
```

Comment:
The variable naming here is a bit confusing, it should be applying the `no-shadow` eslint rule in order to avoid this type of shadowing variables.
```suggestion
      return await rootSpan.traceAsyncFn(async (rootSpan) => {
```

Reply:
:+1: lint is failing. Will go through all changes.

Reply:
It was failing on something else <picture data-single-emoji=":sad_kermit:" title=":sad_kermit:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..NyUjlcispkdud4x2.O8gjKJz8w14yeYq8iedXHcxevHiYCuREFlWgpRx5H7PcB7Y28lWakwvW4PrC6fMHxbnVQj1M3AzDRvgNYtHHBdUMrBL5VM9UFOgw0lQ9KkRw6SAg13NhrrYzV_o.S0lsL9HT_RHC7cKyF-LVaw" alt=":sad_kermit:" align="absmiddle"></picture>

Reply:
in this case, both variables contain the same value, but I fixed it.